omod/src/main/webapp/fragments/dashboardwidgets/discreteValues.gsp

```diff
@@ -0,0 +1,23 @@
+
+<div class="info-section">
+    <div class="info-header">
+        <i class="${app.icon}"></i>
+        <h3>${ ui.message(app.label) }</h3>
+        <a href="${ ui.urlBind("/" + contextPath + app.url, [ "patient.uuid": patient.id ]) }" class="right">
+            <i class="icon-share-alt edit-action" title="Edit"></i>
+        </a>
+    </div>
+    <div class="info-body">
+        <% fields.each { concept, obs -> %>
+            <div>
+                <span style="font-family: 'OpenSansBold'";>${ ui.message(obs.label) }:</span>
+                <% if ( obs.minValue && obs.obsValue && obs.minValue.isNumber() && obs.obsValue.isNumber() &&
+                        (new BigDecimal(obs.obsValue) < new BigDecimal(obs.minValue))) {%>
```

Comment:
Per my comment above, shouldn't we just put the obs in the object, and then do a comparison on obs.valueNumeric?  What is this `isNumber()` method you are calling?  Is that some magic groovy thing?  Does that work?

Basically if `obs.valueNumeric && ...`

Reply:
ok